roster2/src/java/org/sakaiproject/roster/tool/entityprovider/RosterSiteEntityProvider.java

```diff
@@ -165,9 +167,32 @@ public Object getMembership(EntityReference reference, Map<String, Object> param
             data.setMembers(subList);
             data.setMembersTotal(membershipsSize);
 
+            boolean showVisits = sakaiProxy.getShowVisits();
+
+            Map<String, SitePresenceTotal> sitePresenceTotals = new HashMap<String, SitePresenceTotal>();
+            
+            if (showVisits) {
+                sitePresenceTotals = sakaiProxy.getPresenceTotalsForSite(siteId);
+            }
+
+            boolean viewSiteVisits
+                = developerHelperService.isUserAllowedInEntityReference("/user/" + userId
+                                                    , RosterFunctions.ROSTER_FUNCTION_VIEWSITEVISITS
+                                                    , "/site/" + siteId);
+
+            System.out.println("viewSiteVisits: " + viewSiteVisits);
```

Comment:
You don't miss a trick you, do you? I'll take it out. Once, I left the word BALLS in a sysout and Steve S spotted it.